integration/localnet/bootstrap.go

```diff
@@ -1,49 +1,57 @@
 package main
 
 import (
+	"encoding/hex"
 	"encoding/json"
 	"flag"
 	"fmt"
+	"io/ioutil"
 	"os"
 	"path/filepath"
 	"runtime"
 	"time"
 
-	"github.com/plus3it/gorecurcopy"
-	"gopkg.in/yaml.v2"
-
+	"github.com/go-yaml/yaml"
```

Comment:
I'm surprised the linter's not complaining, but the `github.com/onflow/flow-go/*` imports should be separated and last. imports usually are ordered as 
* standard lib
* empty line
* 3rd party
* empty line
* flow-go

Reply:
Sorry about that. I've updated the imports so that github.com/onflow/flow-go/* come last, and are separated from 3rd party imports.